README.md

```diff
@@ -8,6 +8,19 @@ Multiple scenarios will be addressed:
 3. Increasing penetration of variable renewable energy sources.
 4. Add grid flexibility in the form of H2 and thermal storage.
 
+## Instructions to Run TEMOA
+TEMOA is an open source modeling tool available on GitHub (link). Follow the installation instructions here (link). 
+
+After creating a database in sql, navigate to the directory with your database:
```

Comment:
I'm surprised they don't have a utility for generating this SQL database with a simpler input file syntax and use of sqlite3. Nothing to change here... just a comment.